### PR TITLE
SPM-1484: Add instant advisory notifications

### PR DESCRIPTION
**Pull request by michalslomczynski**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by michalslomczynski** [comment]
Maybe `last_updated` column can be dropped. Will be sure after aggregation in notifications-backend.

File: base/models/models.go [hunk 1]
```diff
@@ -139,6 +139,14 @@ type AdvisoryType struct {
 	Preference int
 }
 
+var AdvisoryTypes = map[int]string{
+	0: "unknown",
+	1: "enhancement",
+	2: "bugfix",
+	3: "security",
+	4: "unspecified",
+}
```
Comment (MichaelMraka) [inline base/models/models.go:148]: You should read this enumeration from database not hardcode it.

**Review by MichaelMraka — CHANGES_REQUESTED** [review]

File: base/models/models.go (hunk 1 above)
Comment (michalslomczynski) [reply base/models/models.go:148]: I wanted to discuss this one with you. I've seen we read those values for each request on some endpoint. These values should not change during runtime so wouldn't it be better if we used singleton to read from instead? In similar way as config vars. What do you think @MichaelMraka ?

Comment (michalslomczynski) [reply base/models/models.go:148]: And to be clear reading these from DB for each notification push is not a problem at all, so we can discuss about it later as well if you prefer :wink:

Comment (michalslomczynski) [reply base/models/models.go:148]: Already changed for querying DB for now and lets talk about it later in improvement scope.

**Review by MichaelMraka — APPROVED** [review]

**Pull request merged**